packages/address-book-controller/CHANGELOG.md

```diff
@@ -7,13 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.0]
+
 ### Added
 
 - Add contact event system ([#5779](https://github.com/MetaMask/core/pull/5779))
   - Add `AddressBookControllerContactUpdatedEvent` and `AddressBookControllerContactDeletedEvent` types for contact events
   - Add `list` method on `AddressBookController` to get all address book entries as an array
   - Register message handlers for `list`, `set`, and `delete` actions
   - Add `lastUpdatedAt` property to `AddressBookEntry` to track when contacts were last modified
+- Add `signEip7702Authorization` to `KeyringController` ([#5301](https://github.com/MetaMask/core/pull/5301))
```

Comment:
`KeyringController` isn't in this package. Is there another change worth mentioning here? Perhaps this ought to read

``` markdown
- Bump `@metamask/utils` to `^11.2.0` ([#5301](https://github.com/MetaMask/core/pull/5301))
```

and perhaps this ought to be in the "Changed" section below?